assets/wodby.yml

```diff
@@ -16,3 +16,8 @@ pipeline:
       drush sqlq "UPDATE users_field_data SET pass='#' WHERE mail LIKE '%@ramsalt.com';"
     directory: $WODBY_APP_DOCROOT
     only_if: '[ "$WODBY_ENVIRONMENT_TYPE" = "prod" ] && [ -n "$(drush st --fields=bootstrap)" ]'
+
+  - name: Update localizations
+    type: command
+    command: drush locale:check && drush locale:update
```

Comment:
I have experienced an issue with this. It breaks if the `locale` module is not installed *and* on the first deploys.

We need to add contingency for those two cases before we can merge it _for all sites_.

Reply:
good point, @esolitos do you have any suggestions/workaround for this?